bigquery/README.md

```diff
@@ -0,0 +1,96 @@
+# Overview
+BigQuery interpreter for Apache Zeppelin
+
+# Pre requisities
+You can follow the instructions at [Apache Zeppelin on Dataproc](https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/blob/master/apache-zeppelin/README.MD) to bring up Zeppelin on Google dataproc. 
+You could also install and bring up Zeppelin on Google compute Engine.
+
+# Interpreter Configuration
+
+Configure the following properties during Interpreter creation.
+
+<table class="table-configuration">
+  <tr>
+    <th>Name</th>
+    <th>Default Value</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>zeppelin.bigquery.project_id</td>
+    <td>  </td>
+    <td>Google Project Id</td>
+  </tr>
+  <tr>
+    <td>zeppelin.bigquery.wait_time</td>
+    <td>5000</td>
+    <td>Query Timeout in Milliseconds</td>
+  </tr>
+  <tr>
+    <td>zeppelin.bigquery.max_no_of_rows</td>
+    <td>100000</td>
+    <td>Max result set size</td>
+  </tr>
+</table>
+
+# Connection
+The Interpreter opens a connection with the BigQuery Service using the supplied Google project ID and the compute environment variables.
+
+# Google BigQuery API Javadoc
+[API Javadocs](https://developers.google.com/resources/api-libraries/documentation/bigquery/v2/java/latest/)
```

Comment:
This is JavaDoc for the artefact 

```
<groupId>com.google.apis</groupId>
<artifactId>google-api-services-bigquery</artifactId>
<version>v2-rev265-1.21.0</version>
```
right?

AFAIK it's an open-source library, so would you be so kind to add a link here to it's source code please? This could help future maintainers to keep up with changes, etc.